test/byContract/IexecPoco/06_claim.test.ts

```diff
@@ -0,0 +1,335 @@
+        await iexecPoco
+            .connect(scheduler)
+            .initialize(dealId, taskIndex)
+            .then((tx) => tx.wait());
+        const workerTaskStake = await iexecPoco
+            .viewDeal(dealId)
+            .then((deal) => deal.workerStake.toNumber());
+        const workers = [worker1, worker2];
+        for (const worker of workers) {
+            const { resultHash, resultSeal } = buildResultHashAndResultSeal(
+                taskId,
+                resultDigest,
+                worker,
+            );
+            const schedulerSignature = await buildAndSignContributionAuthorizationMessage(
+                worker.address,
+                taskId,
+                enclaveAddress,
+                scheduler,
+            );
+            await iexecWrapper.depositInIexecAccount(worker, workerTaskStake);
+            await iexecPoco
+                .connect(worker)
+                .contribute(
+                    taskId,
+                    resultHash,
+                    resultSeal,
+                    ethers.constants.AddressZero,
+                    '0x',
+                    schedulerSignature,
+                )
+                .then((tx) => tx.wait());
+        }
+        expect(await iexecPoco.balanceOf(iexecPoco.address)).to.be.equal(
+            dealPrice + schedulerDealStake + workerTaskStake * workers.length,
+        );
+        expect(await iexecPoco.balanceOf(requester.address)).to.be.equal(0);
+        expect(await iexecPoco.frozenOf(requester.address)).to.be.equal(dealPrice);
+        expect(await iexecPoco.balanceOf(scheduler.address)).to.be.equal(0);
+        expect(await iexecPoco.frozenOf(scheduler.address)).to.be.equal(schedulerDealStake);
+        for (const worker of workers) {
+            expect(await iexecPoco.balanceOf(worker.address)).to.be.equal(0);
+            expect(await iexecPoco.frozenOf(worker.address)).to.be.equal(workerTaskStake);
+        }
+        expect(await iexecPoco.balanceOf(kittyAddress)).to.be.equal(0);
+        expect(await iexecPoco.frozenOf(kittyAddress)).to.be.equal(0);
+        expect((await iexecPoco.viewTask(taskId)).status).to.equal(TaskStatusEnum.REVEALING);
+        await time.setNextBlockTimestamp(startTime + 10 * categoryTime);
+
+        const claimReceipt = await iexecPoco
+            .connect(anyone)
+            .claim(taskId)
+            .then((tx) => tx.wait());
+        expect(claimReceipt)
+            .to.emit(iexecPoco, 'Transfer')
+            .withArgs(iexecPoco.address, requester.address, taskPrice)
+            .to.emit(iexecPoco, 'Unlock')
+            .withArgs(requester.address, taskPrice)
+            .to.emit(iexecPoco, 'Seize')
+            .withArgs(scheduler.address, schedulerTaskStake, taskId)
+            .to.emit(iexecPoco, 'Reward')
+            .withArgs(kittyAddress, schedulerTaskStake, taskId)
+            .to.emit(iexecPoco, 'Transfer')
+            .withArgs(iexecPoco.address, kittyAddress, schedulerTaskStake)
```

Comment:
```suggestion
            .to.emit(iexecPoco, 'Transfer')
            .withArgs(iexecPoco.address, kittyAddress, schedulerTaskStake)
            .to.emit(iexecPoco, 'Reward')
            .withArgs(kittyAddress, schedulerTaskStake, taskId)
```
To keep a consistent order.